libknary/util.go

```diff
@@ -258,7 +262,8 @@ func CheckTLSExpiry(domain string) bool {
 		Printy(certMsg, 2)
 		logger("WARNING", certMsg)
 		go sendMsg(":lock: " + certMsg)
+		return false, nil
```

Comment:
Should this not be a `return true, nil` because there is no error, the function works as expected?

Reply:
This function may need some reworking